book/src/testing-drivers.md

```diff
@@ -3,4 +3,5 @@
 This section describes how CSI developers can test their CSI drivers.
 
 * [Unit Testing](unit-testing.md)
-* [Functional Testing](functional-testing.md)
\ No newline at end of file
+* [Functional Testing](functional-testing.md)
```

Comment:
The functional page was intended for the K8s e2es I believe. Can we merge the two pages somehow?

Reply:
Yes it does. Should I copy over e2e-tests.md to this page?

Reply:
I merged the two pages and squashed the commits.